tests/virtualization/external/prepare.pm

```diff
@@ -29,7 +29,12 @@ sub run {
             assert_script_run(qq(echo "$ip $guest" >> /etc/hosts));
         }
     } else {
-        assert_script_run "echo \"\$(dig +short $virt_autotest::common::guests{$_}->{ip}) $_ # virtualization\" >> /etc/hosts" foreach (keys %virt_autotest::common::guests);
+        foreach my $guest (keys %virt_autotest::common::guests) {
+            my $vm_name = $virt_autotest::common::guests{$guest}->{vm_name};
+            my $ip = script_output(qq(ssh -o StrictHostKeyChecking=no Administrator\@win2k19.qa.suse.cz 'powershell "get-vm -Name $vm_name | select -ExpandProperty networkadapters | select ipaddresses"' | awk -F '[{,]' '{print \$2}'));
```

Comment:
It this OK to match IP address, no blank row for the result.

ssh -o StrictHostKeyChecking=no Administrator\@win2k19.qa.suse.cz 'powershell "get-vm -Name sles-12.3_openQA-virtualization-maintenance | select -ExpandProperty networkadapters | select ipaddresses"' | grep -oE '[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+

10.100.101.47

Reply:
grep will work too in this case and remove the empty lines